common/persistence/sql/sql_test_utils.go

```diff
@@ -192,8 +192,8 @@ func getCadencePackageDir() (string, error) {
 	if err != nil {
 		panic(err)
 	}
-	cadenceIndex := strings.LastIndex(cadencePackageDir, "/cadence/")
-	cadencePackageDir = cadencePackageDir[:cadenceIndex+len("/cadence/")]
+	cadenceIndex := strings.LastIndex(cadencePackageDir, "cadence/")
```

Comment:
This broke my setup with a folder named /my-cadence/.
I've copied the solution from nosql version